website/docs/configuration-options.md

```diff
@@ -743,6 +743,29 @@ The above will set `rangeStrategy` to `replace` for any package starting with `a
 
 ### paths
 
+### prPriority
```

Comment:
Remove section and move sample to other `psPriority` section

Reply:
i added to the wrong section before and i think ###prPriority should be under ##PackageRules

Reply:
Then you have to add
```
stage: 'package',
parent: 'packageRules',
```

to your definition, update schema and move config options section to `PackageRules`

Reply:
See my other comment - I think leave it outside packageRules

Reply:
> Then you have to add
> 
> ```
> stage: 'package',
> parent: 'packageRules',
> ```
> 
> to your definition, update schema and move config options section to `PackageRules`

@ViceIce you are right. ##prPriority will be outside of ##packageRules.